[ 2025-03-29 16:33:48,116 ] 125 faiss.loader - INFO - Loading faiss with AVX2 support.
[ 2025-03-29 16:33:48,370 ] 127 faiss.loader - INFO - Successfully loaded faiss with AVX2 support.
[ 2025-03-29 16:33:48,380 ] 185 faiss - INFO - Failed to load GPU Faiss: name 'GpuIndexIVFFlat' is not defined. Will not load constructor refs for GPU indexes.
[ 2025-03-29 16:41:25,430 ] 125 faiss.loader - INFO - Loading faiss with AVX2 support.
[ 2025-03-29 16:41:25,540 ] 127 faiss.loader - INFO - Successfully loaded faiss with AVX2 support.
[ 2025-03-29 16:41:25,547 ] 185 faiss - INFO - Failed to load GPU Faiss: name 'GpuIndexIVFFlat' is not defined. Will not load constructor refs for GPU indexes.
[ 2025-03-29 16:41:50,649 ] 125 faiss.loader - INFO - Loading faiss with AVX2 support.
[ 2025-03-29 16:41:50,755 ] 127 faiss.loader - INFO - Successfully loaded faiss with AVX2 support.
[ 2025-03-29 16:41:50,761 ] 185 faiss - INFO - Failed to load GPU Faiss: name 'GpuIndexIVFFlat' is not defined. Will not load constructor refs for GPU indexes.
[ 2025-03-29 16:42:18,781 ] 125 faiss.loader - INFO - Loading faiss with AVX2 support.
[ 2025-03-29 16:42:18,836 ] 127 faiss.loader - INFO - Successfully loaded faiss with AVX2 support.
[ 2025-03-29 16:42:18,866 ] 185 faiss - INFO - Failed to load GPU Faiss: name 'GpuIndexIVFFlat' is not defined. Will not load constructor refs for GPU indexes.
[ 2025-03-29 16:42:21,218 ] 65 langchain_google_genai.chat_models - WARNING - Retrying langchain_google_genai.chat_models._chat_with_retry.<locals>._chat_with_retry in 2.0 seconds as it raised ResourceExhausted: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, head to: https://ai.google.dev/gemini-api/docs/rate-limits. [violations {
}
, links {
  description: "Learn more about Gemini API quotas"
  url: "https://ai.google.dev/gemini-api/docs/rate-limits"
}
, retry_delay {
  seconds: 40
}
].
[ 2025-03-29 16:42:23,506 ] 14 root - INFO - Error occured in python script name at G:\AI MODELS\Portfolio\src\components\model.py on line number 58 error messgae 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, head to: https://ai.google.dev/gemini-api/docs/rate-limits. [violations {
}
, links {
  description: "Learn more about Gemini API quotas"
  url: "https://ai.google.dev/gemini-api/docs/rate-limits"
}
, retry_delay {
  seconds: 37
}
]
